### PR TITLE
Updating Documentation for final product

### PR DESCRIPTION
**Pull request by cjinting-nus**
Includes UG/DG/PPP

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -18,25 +18,32 @@ By: `W13-1`      Since: `Sept 2018`      Licence: `MIT`
...
+image::StartUpScreen.png[width="790"]
 +
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
+.  If it is the first time you are using the app, follow the instructions on-screen to create your account and press kbd:[Enter] to execute it. +
+****
+Creating account example: +
+
+`register Paul 123` +
```
Comment (cjinting-nus) [inline docs/UserGuide.adoc]: This is with consideration that the fields need not be all filled in for v1.4 @bennchong

Comment (bennchong) [reply docs/UserGuide.adoc]: @nianfei97 @alexiscatnip. Just an update, for registering there will be prefixes to each argument, eg. register u/Paul pw/123. You can just push this first, I will make it a note to edit the UG as needed

Comment (cjinting-nus) [reply docs/UserGuide.adoc]: Noted with thanks

**Comment by cjinting-nus** [comment]
That is a really weird bug.. I'll fix it and push again

**Pull request merged**